golden_response/staging/query/weather1.json

```diff
@@ -6,7 +6,7 @@
     {
       "cells": [
         {
-          "value": "-2.9344179732812004"
+          "value": "-2.934417973281198"
```

Comment:
have no idea why there is precision changes

Reply:
my guess is something upstream in the weather data changed. since the diff is negligible and weather is not really used by anyone, this is ok for me.